### PR TITLE
Clean up visitor pattern in Title.cpp

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: src/OpenLoco/src/Title.cpp [hunk 1]
```diff
@@ -187,6 +176,51 @@ namespace OpenLoco::Title
         Audio::stopMusic();
     }
 
+    static void handleStep([[maybe_unused]] const WaitStep& step)
```
Comment (ZehMatt) [inline src/OpenLoco/src/Title.cpp:179]: The argument is actually used here.

Comment (AaronVanGeffen) [reply src/OpenLoco/src/Title.cpp:179]: Ah, you're right!

**Comment by duncanspumpkin** [comment]
Could do with apply this same setup to the intro (not that many care about the intro)

**Comment by AaronVanGeffen** [comment]
> Could do with apply this same setup to the intro (not that many care about the intro)

Probably, though that will require quite a refactor. Better to file an issue for it, I think. Could be a 'good first issue' for someone new to pick up.

**Review by ZehMatt — APPROVED** [review]

**Pull request merged**